src/tools/tidy/src/ui_tests.rs

```diff
@@ -1,12 +1,14 @@
 //! Tidy check to ensure below in UI test directories:
 //! - there are no stray `.stderr` files
 
-use std::collections::BTreeSet;
+use std::collections::{BTreeSet, HashMap};
 use std::ffi::OsStr;
 use std::fs;
 use std::io::Write;
 use std::path::{Path, PathBuf};
 
+use ignore::Walk;
```

Comment:
This doesn't seem related to a conflict?

Reply:
it is somewhat, because it was removed in a commit/pr that caused the conflict

Reply:
was unsure how to represent the change... whether this commit, or one that fixes the rest of build (the child)